cpp/src/io/parquet/page_enc.cu

```diff
@@ -2926,6 +2926,9 @@ __device__ uint8_t* EncodeStatistics(uint8_t* start,
     auto const [min_ptr, min_size] =
       get_extremum(&s->min_value, dtype, scratch, true, NO_TRUNC_STATS);
     encoder.field_binary(6, min_ptr, min_size);
+    // cudf min/max statistics are always exact (i.e. not truncated)
+    encoder.field_bool(7, true);
+    encoder.field_bool(8, true);
```

Comment:
Hmm...good point. As currently written, the stats encoding will always write _something_ in the min and max fields if `s->has_minmax` is true, so I think it's safe to always set the `exact` fields to true here.

That said, the rules around when min/max should have values encoded and not have been in flux (and as with everything Parquet, are not well documented), so it would probably be a good idea to make sure we're following all of the rules properly. I'm not sure if that's in scope for this PR or should be a separate issue.